packages/ui/src/i18n/dictionaries/authenticator/defaultTexts.ts

```diff
@@ -19,6 +19,7 @@ export const defaultTexts = {
   ENTER_USERNAME: 'Enter your username',
   FAMILY_NAME: 'Family Name',
   GIVEN_NAME: 'Given Name',
+  FORGOT_PASSWORD: 'Forgot Password?',
```

Comment:
Added a "short" version to improve the layout of the secondary buttons in the `SignIn` component:

![image](https://user-images.githubusercontent.com/15002885/202590630-42628a50-80b7-4276-b9ab-3ec7e54ff826.png)